static-site/content/blog/2024-06-18-oropouche-analysis-and-resources.md

```diff
@@ -0,0 +1,69 @@
+---
+author: "Miguel Paredes, James Hadfield, John Anderson, Jover Lee, Trevor Bedford"
+date: "2024-10-18"
+title: "Phylogenetic analysis of Oropuche Virus"
```

Comment:
not sure what the prevailing style is across current blog posts, but I would suggest either title-casing the whole thing, or down-casing `"Virus"`...

Reply:
agreed! I updated this in the most recent commit, and suggestion actually highlighted out to me that "Oropouche" is mispelled above so I went ahead and fixed that. I also added in an intro paragraph as per your suggestion